portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,93 @@ function addFunFact() {
+    document.getElementById('continue').style.display = "none";
+}
+
+/*
+ * Stops the slideshow.
+ */
+function stop() {
+    clearInterval(timer); // Stops the automatic changing of the image and description.
+    document.getElementById('continue').style.display = "block";
+}
+
+// let image_sources = ['/images/1.jpg', '/images/2.jpg', '/images/3.jpg', '/images/4.jpg']
+
+// let descriptions = ['Family has always been something really important to me, especially my cousins. ' + 
+//                     'Every year, we try to get together in what we call \'Cuz Unite\'. During Cuz Unites, ' +
+//                     'we go on wild adventures, play games, and just spend time catching up and enjoying ' +
+//                     'each other\'s company. Around my cousins I always feel loved and free to be myself.',
+    
+//                     'At Rice, students are randomly split into 11 residential colleges, which will serve as an ' +
+//                     'inclusive community throughout their four years on campus (and beyond). My college is ' +
+//                     'Wiess college, and from the moment I was welcomed during orientation week it has felt ' + 
+//                     'like home. From the friendly faces I see in commons every day to our quirky and exciting ' +
+//                     'traditions, I am truly grateful for my Team Family Wiess!', 
+
+//                     'The summer after my senior year of high school, I went on an epic month-long European vacation ' +
+//                     'with my family, where we visited The Netherlands, Belgium, France, Scotland, and England. It was ' +
+//                     'an amazing experience to get to see and explore so many different places, and I am so ' + 
+//                     'appreciative of my family for making it happen. Hopefully I can continue to travel the world ' +
+//                     'on my own someday.',
+                    
+//                     'Ever since I was really little and just twirling around the house with scarves, ' + 
+//                     'I\'ve always had a passion for dance. Whether it was performing on stage with my ' +
+//                     'competitive team, at a rally with my high school class, or in the quad with my kpop ' +
+//                     'dance club, I had so much fun expressing myself through movement. Sometimes the ' +
+//                     'competitive nature of the dance world got to me, but I would turn back to low-pressure ' + 
+//                     'dance environments with friends to remind me of the happiness it brought me.'];
+
+/*
+ * Changes the image and description on the page.
+ */ 
+// function changeDisplay() {
+//     if (count == 3) {
+//         count = 0;
+//     } else {
+//         count ++;
+//     }
+//     document.getElementById('img').src = image_sources[count];
+//     document.getElementById('description').innerText = descriptions[count];
+// }
```

Comment:
Recommend getting rid of all the dead code.

Reply:
Done! (see commit "Address requested changes...")